db/views/reporting_facility_state_dimensions_v01.sql

```diff
@@ -0,0 +1,84 @@
+WITH monthly_registration_patient_states AS 
```

Comment:
This is not a new view, its just renaming the previous `reporting_facility_state_groups` to `reporting_facility_state_dimensions`.  Names are hard, and `group` was just way too general and is in use for other meanings in our app.